### PR TITLE
support for QUITMSG1..QUITMSG14 in Dehacked

### PR DESCRIPTION
**Pull request by rfomin**
Test WAD: [D2ISOv2.wad](https://www.doomworld.com/idgames/levels/doom2/megawads/d2isov2) Also works in ZDoom: https://zdoom.org/wiki/Strings

### CODE REVIEW

File: src/m_menu.c [hunk 1]
```diff
@@ -7005,18 +7005,18 @@ void M_Init(void)
   // [crispy] remove DOS reference from the game quit confirmation dialogs
   {
     const char *platform = SDL_GetPlatform();
-    const char *string;
```
Comment (fabiangreffrath) [inline src/m_menu.c:7008]: So, the OS replacement gimmicks still work?

Comment (rfomin) [reply src/m_menu.c:7008]: Yes:
image

**Review by fabiangreffrath — APPROVED** [review]

File: src/d_deh.c [hunk 2]
```diff
@@ -123,6 +123,20 @@ char *s_D_CDROM     = D_CDROM;
 char *s_PRESSKEY    = PRESSKEY;
 char *s_PRESSYN     = PRESSYN;
 char *s_QUITMSG     = QUITMSG;
+char *s_QUITMSG1    = QUITMSG1;
```
Comment (fabiangreffrath) [inline src/d_deh.c:126]: I wonder if there is a preprocessor macro that one could use in a loop? Do you know of something like that?

Comment (rfomin) [reply src/d_deh.c:126]: It's a pain: https://stackoverflow.com/questions/319328/how-to-write-a-while-loop-with-the-c-preprocessor C preprocessor is for simple things 🤓

Comment (fabiangreffrath) [reply src/d_deh.c:126]: 🤣 no way.

**Pull request merged**